CMakeLists.txt

```diff
@@ -1143,13 +1137,9 @@ configure_file("${hpx_SOURCE_DIR}/cmake/templates/hpxrun.py.in"
 # installation instructions
 ################################################################################
 install(
-  DIRECTORY ${CMAKE_BINARY_DIR}/bin/
-  DESTINATION hpx/bin/
-  COMPONENT core
-  FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
-                   GROUP_READ GROUP_EXECUTE
-                   WORLD_READ WORLD_EXECUTE
-  FILES_MATCHING PATTERN "hpxcxx" EXCLUDE)
+  FILES "${CMAKE_BINARY_DIR}/bin/hpxrun.py"
+  DESTINATION bin
+  COMPONENT core)
 
```

Comment:
> The previous code did not install anything, so I deleted it. The new code installs 
> hpxrun.py which I need for the LGD test.

Sure, what I meant is why did you remove the `FILE_PERMISSIONS` clause? Shouldn't scripts be made executable?